src/Action/Iterator.php

```diff
@@ -132,14 +237,14 @@ public function order($fields)
     /**
      * Limit Iterator.
      *
-     * @param int $cnt
-     * @param int $shift
+     * @param int $length
+     * @param int $offset
      *
      * @return $this
      */
-    public function limit($cnt, $shift = 0)
+    public function limit($length, $offset = 0)
```

Comment:
count, right?

Reply:
We limit the length of the result. The same name is used in PHP native functions as well (e.g. see `array_slice`)

Reply:
this is cosmetic change only, I did more research, `$limit` is the right name :)

Reply:
Cosmetics is all important. See what girls do with themselves 😉 
Changed now.